src/app/content/highlights/components/cardUtils.ts

```diff
@@ -9,13 +9,26 @@ import { assertDefined } from '../../../utils/assertions';
 import { cardMarginBottom } from '../constants';
 import { HighlightData } from '../types';
 
+export const hiddenHighlightOffset = -9999;
+
 export const getHighlightOffset = (container: HTMLElement | undefined, highlight: Highlight) => {
   if (!container || !highlight.range || !highlight.range.getBoundingClientRect) {
     return;
   }
 
   const {top, bottom } = highlight.range.getBoundingClientRect();
 
+  const highlightElement = highlight.elements[0] as HTMLElement;
```

Comment:
i think this might be undefined, what is the type that you get out of the library? is it not already HTMLElement?

Reply:
it has `elements: HTMLElement[];` type.
However without the casting `HTMLElement` (imported from `'openstax/types/lib.dom`), ts throws error: `Property 'closest' does not exist on type 'HTMLElement'.`.